page-object-code-gen/src/main/java/com/kaspresso/components/pageobjectcodegen/SupportedViews.kt

```diff
@@ -0,0 +1,6 @@
+package com.kaspresso.components.pageobjectcodegen
+
+object SupportedViews {
+    val collectableElements = listOf("android.widget.Button", "android.widget.TextView", "android.widget.ImageView")
+    val elementsWithChild = listOf("androidx.recyclerview.widget.RecyclerView")
```

Comment:
Why only recycler view? Are there any problems with the scroll view, pager, horizontal scroll view etc.?